sympy/ntheory/tests/test_residue.py

```diff
@@ -263,3 +263,11 @@ def test_residue():
     assert quadratic_congruence(5, 10, 14, 2) == [0]
     assert quadratic_congruence(10, 17, 19, 2) == [1]
     assert quadratic_congruence(10, 14, 20, 2) == [0, 1]
+
+    assert factmod(6, 11) == 5
```

Comment:
It would be good to test against the actual value here, like

```py
assert facmod(6, 11) == factorial(6) % 11 == 5
```

Reply:
I'll add them. Thanks.